rcl/test/rcl/test_subscription.cpp

```diff
@@ -133,16 +133,14 @@ TEST_F(CLASSNAME(TestSubscriptionFixture, RMW_IMPLEMENTATION), test_subscription
     rcl_ret_t ret = rcl_publisher_fini(&publisher, this->node_ptr);
```

Comment:
Now that we've created separate tests, we don't need the publisher in this test.

Reply:
Thanks. It's by b5838ee5c790de4544312e2ce6ac85f222628f89.